docs/UserGuide.md

```diff
@@ -190,7 +190,13 @@ _Details coming soon ..._
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
 
 --------------------------------------------------------------------------------------------------------------------
-
+## Adding contacts to folder
+To organize and group contacts into an arbitrary folder
+Format: `echo CONTACT_INDEX >> FOLDER_NAME`
+- `CONTACT_INDEX` must be a positive integer 1, 2, 3, ...
+- `FOLDER_NAME` must be an existing folder
+    - Command fails if there is no existing folder with that name
+    
```

Comment:
i think this should be after the **Creating a group: `mkdir`** section and before the **Saving the data** section, as a H3 header